tools/generator/CodeGenerator.cs

```diff
@@ -26,7 +26,12 @@ public static int Main (string[] args)
 			if (options == null)
 				return 1;
 
-			Run (options);
+			try {
+				Run (options);
+			} catch (BindingGeneratorException) {
+				return 1;
+			}
```

Comment:
Should other exception types be handled, "just in case", so that we don't get the "unhandled exception" output?  Or do we *want* the unhandled exception output?

Reply:
Makes sense, added.